indico/modules/events/abstracts/controllers/display.py

```diff
@@ -76,3 +81,26 @@ def _process(self):
         sorted_abstracts = sorted(self.abstracts, key=attrgetter('friendly_id'))
         pdf = AbstractsToPDF(self.event_new, sorted_abstracts)
         return send_file('my-abstracts.pdf', pdf.generate(), 'application/pdf')
+
+
+class RHSubmitAbstract(RHMyAbstractsBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            if isinstance(data['submitted_for_tracks'], Track):
+                data['submitted_for_tracks'] = {data['submitted_for_tracks']}
+            elif data['submitted_for_tracks'] is None:
+                data['submitted_for_tracks'] = set()
+            abstract = create_abstract(self.event_new, *get_field_values(data))
+            flash(_("Your abstract with title '{}' has been successfully submitted. It is registered with the number "
```

Comment:
I only used our old message, I am open to other suggestions. Is it really "with **the** title"? It sounds fine to me to say "Your abstract with title", maybe because of Greek! :yum: